draft-ietf-rats-architecture.md

```diff
@@ -138,20 +138,35 @@ informative:
 
 --- abstract
 
-In network protocol exchanges it is often the case that
-one entity requires believable evidence about the operational state of a remote
-peer.  Such evidence is typically conveyed as claims
-about the peer's software and hardware platform, and is subsequently appraised in order
-to assess the peer's trustworthiness.  The process of generating and
-appraising this kind of evidence is known as remote
-attestation.  This document describes an architecture for remote attestation
-procedures that generate, convey, and appraise evidence about a peer's
-operational state.
+In network protocol exchanges it is useful for one end of a
+communication to know if the other end is in a proper operating state.
```

Comment:
```suggestion
communication to know whether the other end is in a proper operating state.
```
(https://grammarist.com/grammar/if-vs-whether/ says "if" is considered informal, though the meaning is the same, so to me "whether" reads better)